rsmetrics.py

```diff
@@ -207,10 +210,19 @@ def inner():
 # used to create the data frames. _ids are filtered out from the column
 # results during the query
 logging.info("Reading users...")
+
 run.users = pd.DataFrame(
-    list(rsmetrics_db["users"].find({"provider": {"$in": [args.provider]}},
-                                    {"_id": 0}))
+    list(rsmetrics_db["users"].find({
```

Comment:
In RS Metrics the current algorithm for choosing which users (or services) to select for the metrics calculations is:


- the user (or service) must belong to Cyfronet or Athena provider's dataset
- if the created_on and the deleted_on are not set (=None), use that user (or service)
- if the created_on is less than or equal to the endtime of the desired time range, use that user (or service)
- if the deleted_on is greater than or equal to the starttime of the desired time range, use that user (or service)